bigtable/client/metadata_update_policy.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/metadata_update_policy.h"
+#include "bigtable/client/internal/make_unique.h"
+
+#include <sstream>
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+const MetadataParamTypes MetadataParamTypes::PARENT("parent");
```

Comment:
s/const MetadataParamTypes/MetadatParamTypes const/ for consistency with the rest of the code.